client/core/trade.go

```diff
@@ -211,8 +211,14 @@ func (t *trackedTrade) coreOrder() *Order {
 // should be called with the mtx >= RLocked.
 func (t *trackedTrade) coreOrderInternal() *Order {
 	corder := coreOrderFromTrade(t.Order, t.metaData)
+
 	corder.Epoch = t.dc.marketEpoch(t.mktID, t.Prefix().ServerTime)
-	corder.LockedAmt = t.lockedAmount()
+
+	if t.coinsLocked {
+		corder.LockedAmt = sumCoinMap(t.coins)
+	} else if t.changeLocked {
+		corder.LockedAmt = t.change.Value()
+	}
```

Comment:
Since we're keeping `(*trackedTrade).lockedAmount` because it's used by `(*Core).lockedAmounts` why not keep using it here?